app/adminPage.html

```diff
@@ -78,8 +78,9 @@ <h4 class="search-heading">Filter Results</h4>
                                     <select name="filterScorePercentage" id="filterScorePercentage" class="dropdown">
                                         <option value="" disabled selected>Score Percentage</option>
                                         <option value="3">-none-</option>
-                                        <option value="1">+70%</option>
-                                        <option value="2">-69%</option>
+                                        <option value="2">&lt;70%</option>
+                                        <option value="1">70% - 96%</option>
+                                        <option value="4">&#8805;97%</option>
```

Comment:
do these need to stay in this value order? just a bit messy

Reply:
infact, are these the options we want? or is it better to have all, 70+ (including 97+), and 97+